commands/trigger.go

```diff
@@ -548,17 +623,30 @@ func (t *Trigger) Update(Client *whisk.Client, args []string) error {
 
 	if err != nil && httpResp.StatusCode == 404 {
 		t.Create(Client, args)
+		return nil
 	} else if err != nil {
 		whisk.Debug(whisk.DbgError, "Client.Triggers.Get(%s) failed: %s\n", qualifiedName.GetEntityName(), err)
 		errStr := wski18n.T("Unable to get trigger '{{.name}}': {{.err}}",
 			map[string]interface{}{"name": qualifiedName.GetEntityName(), "err": err})
 		werr := whisk.MakeWskErrorFromWskError(errors.New(errStr), err, whisk.EXIT_CODE_ERR_GENERAL, whisk.DISPLAY_MSG, whisk.NO_DISPLAY_USAGE)
 		return werr
-	} else {
+	}
 
-		// Get full feed name from trigger get request as it is needed to get the feed
-		if retTrigger != nil && retTrigger.Annotations != nil {
-			fullFeedName = getValueString(retTrigger.Annotations, "feed")
+	// Get full feed name from trigger get request as it is needed to get the feed
```

Comment:
This this update method has 200 lines of code making it extremely hard to understand quickly.